source/code/source/playfab/PlayFabAndroidHttpPlugin.cpp

```diff
@@ -456,7 +456,7 @@ namespace PlayFab
         SetPredefinedHeaders(requestTask);
 
         // Call SetHeader
-        auto headers = requestContainer.GetRequestHeaders();
+        const auto& headers = requestContainer.GetRequestHeaders();
```

Comment:
>auto [](start = 14, length = 4)

it's not obvious what this type is from the right-hand side which seems to contradict your auto rules. I agree with those rules and think the full type is better here (similarly for GetCallback)